src/assemble_workflow/bundle_linux_deb.py

```diff
@@ -126,6 +126,8 @@ def build(self, name: str, dest: str, archive_path: str, build_cls: BuildManifes
         subprocess.check_call(bundle_cmd, cwd=ext_dest, shell=True)
 
         # Move artifact to repo root before being published to {dest}
-        # for dirpath, dirnames, filenames in os.walk(os.path.join('/tmp/opensearch*')):
+        # In debuild, the final package is created in the parent directory of the sources
+        # https://github.com/opensearch-project/opensearch-build/issues/4532#issuecomment-2091726868
+        bundle_artifact_path = f"/tmp/{self.filename}_{deb_version}_{deb_architecture(build_cls.architecture)}.deb"
```

Comment:
Is this path the same as before?

Reply:
It is same, but in previous implementation the var `bundle_artifact_path` is missing assignment other than `None`, resulted in:

```
2024-03-14 17:05:15 INFO     Found deb file: None
```

Now:
```
2024-05-02 21:50:15 INFO     Found deb file: /tmp/opensearch_2.14.0_amd64.deb
```

Thanks.

Reply:
Make sense. I see it doesn't assign any value after line 100. Thanks @peterzhuamazon